torch/_inductor/codegen/wrapper.py

```diff
@@ -959,6 +961,9 @@ def __init__(self):
         self.supports_intermediate_hooks = False
         self.outputs_need_copy = set()
         self.kernel_callsite_id = count()
+        self.int_array_id = count()  # for int array local variable declarations
```

Comment:
You actually don't need the temp arrays, because you're in C++ and an initializer list `{1, 2}` will cast into a `const int*` just fine